src/jobs/run.yml

```diff
@@ -2,16 +2,17 @@ description: >
   A single, complete job to run Cypress end-to-end tests in your application.
 # What will this job do?
 executor: default
+parallelism: << parameters.parallelism >>
 
 parameters:
+  install-command:
+    description: Overrides the default NPM command (npm ci)
+    type: string
+    default: ""
```

Comment:
Can you recap why this wasn't working as steps? I didn't get clear notes on that